client/src/components/Buttons.js

```diff
@@ -80,3 +80,9 @@ export const ExitInterviewButton = (props) => {
   const { text } = props
   return <Button variant="outlined" className={classes.exitInterviewButton}>{text}</Button>
 }
+
+export const AnswerButton = (props) => {
+  const classes = useStyles()
+  const { text } = props
+  return <Button variant="outlined" className={classes.answerButton}>{text}</Button>
```

Comment:
I feel like these buttons are not really reusable and having looked at this more closely in CSS, there is a lot of repetition in the CSS. 

I would have rather had more general buttons with some props that make it customizable (like colors, etc.)

I think this is too involved to refactor now, but it's just some feedback to note